test/HydrationRepository-test.js

```diff
@@ -52,6 +53,9 @@ describe('HydrationRepository', () => {
   });
 
   it('should return how many fluid oz they consumed for a specific day', () => {
+    chai.spy.on(hydrationRepository, 'getUserInfoByDate', () => {
+      return { userID: 12, date: "2019/08/22", numOunces: 58 }
+    })
     expect(hydrationRepository.totalOzDay(12, '2019/08/22', 'hydrationData')).to.equal(58);
   });
 
```

Comment:
Do we also need to add the expect statement for the spies? like:
expect(getUserInfoByDate).to.have.been.called?